airsonic-main/src/main/resources/org/airsonic/player/theme/default_light.properties

```diff
@@ -1,57 +1,61 @@
-
 # Icons used by light themes.
 # Author: Sindre Mehus
 
-addImage = icons/default_light/add.png
-addOverlayImage = icons/default_light/add_overlay.png
-addNextImage = icons/default_light/add_next.png
-androidImage = icons/default_light/android.png
-backImage = icons/default_light/back.png
-castActiveImage = icons/default_light/cast_active.png
-castIdleImage = icons/default_light/cast_idle.png
-castPauseImage = icons/default_light/cast_pause.png
-castPlayImage = icons/default_light/cast_play.png
-clearRatingImage = icons/default_light/remove.png
-currentImage = icons/default_light/current.png
-donateImage = icons/default_light/donate.png
-donateSmallImage = icons/default_light/donate_small.png
-downImage = icons/default_light/down.png
-downloadImage = icons/default_light/download.png
-forwardImage = icons/default_light/forward.png
-helpImage = icons/default_light/help.png
-helpPopupImage = icons/default_light/help_small.png
-homeImage = icons/default_light/home.png
+addImage = icons/default_light/add.svg
+addOverlayImage = icons/default_light/add_overlay.svg
+addNextImage = icons/default_light/add_next.svg
+androidImage = icons/default_light/android.svg
+backImage = icons/default_light/back.svg
+castActiveImage = icons/default_light/cast_active.svg
+castIdleImage = icons/default_light/cast_idle.svg
+castPauseImage = icons/default_light/cast_pause.svg
+castPlayImage = icons/default_light/play.svg
+clearRatingImage = icons/default_light/remove.svg
+currentImage = icons/default_light/current.svg
+donateImage = icons/default_light/donate.svg
+donateSmallImage = icons/default_light/donate.svg
+downImage = icons/default_light/down.svg
+downloadImage = icons/default_light/download.svg
+forwardImage = icons/default_light/forward.svg
+helpImage = icons/default_light/help.svg
+helpPopupImage = icons/default_light/help.svg
+homeImage = icons/default_light/home.svg
 html5Image = icons/default_light/html5.png
 keyboardImage = icons/default_light/keyboard.png
-logImage = icons/default_light/log.png
+logImage = icons/default_light/log.svg
 logoImage = icons/default_light/logo.png
```

Comment:
You can find the logo in svg if you like, check the contrib folder.

Reply:
I had some alignment issues, I'll send a separate PR for this.